src/main/java/org/minions/devfund/angela/battleship/Ocean.java

```diff
@@ -0,0 +1,177 @@
+package org.minions.devfund.angela.battleship;
+
+import java.util.Random;
+
+/**
+ * Manages ship's ocean.
+ */
+public class Ocean {
+    private static final int OCEAN_SIZE = 20;
+    private static final int SHIPS = 13;
+    private Ship[][] ships;
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "ships" to prevent any misunderstanding/clash with field "SHIPS" defined on line 10. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1845)